handlers/preview.go

```diff
@@ -63,17 +67,21 @@ func (f *Filter) PreviewPage(w http.ResponseWriter, req *http.Request) {
 	filterID := fj.Links.FilterBlueprint.ID
 
 	var dimensions []filter.ModelDimension
-	for _, header := range prev.Headers {
-		dimensions = append(dimensions, filter.ModelDimension{Name: header})
+	for i, header := range prev.Headers {
+		if even(i) {
```

Comment:
This can't just check for even columns, as the observation and metadata may take up any number of columns (not just the first). The `V4_X` data can be used to determine how many additional columns are present to offset before taking every other column